### ISSUE TITLE
support for large V3

### ISSUE THREAD

**Issue by andymees** [octimot/StoryToolkitAI#140]
If possible, it would be great to have support for the Whisper large-v3 model... shows improved performance over a wide variety of languages, showing 10% to 20% reduction of errors compared to Whisper large-v2; adds support for Cantonese.

**Comment by octimot** [comment]
This will be taken care of in the next days!

Cheers

**Comment by octimot** [comment]
I've just been playing with this and, at least from my tests v3 seems to be missing a lot of chunks of tests compared to v2 for english transcriptions - especially on short audio files.

But, I'll push the update as soon as possible.

**Comment by pantau000** [comment]
I'd be happy to run a test for Portuguese.

**Comment by ryzen88** [comment]
Would it not be more practical of having a folder where we can just download new models in from hugging face for example?
Seems a lot of work if you have to update the tool for integrating a sort of the same, just only new iteration of a model. 
For example i play around with kolboldai that uses LLM for text generation but it can handle lots of models not in there by default simple by downloading the model form huggingface and placing it in the model folder.
Maybe in the near future other players come with better models than this whisper from openai.

I have no coding skills so probably its way more complex then i think.

**Comment by octimot** [comment]
@ryzen88 

Hopefully we can implement that in the future, but right now the tool is using the actual [openai/whisper](https://github.com/openai/whisper) package to list the available models. 

In the meantime, we've also extended the default whisper transcribe file (see [mots_whisper.py](https://github.com/octimot/StoryToolkitAI/blob/main/storytoolkitai/integrations/mots_whisper.py)) that allows us to queue, cancel and do all sorts of useful operations during and after the transcription process.

The bottom line is that whenever openai release a new model, we can only use it by updating the whisper package and that needs to be manually checked by us so that it doesn't interfere with anything else we've extended on top of it.

Anyway, stay tuned... the push is coming soon!

**Comment by octimot** [comment]
I just pushed an update (0.19.6) which adds support for large v3.

@pantau000 ready for some tests? :-)

Cheers!
